README.md

```diff
@@ -1 +1,11 @@
-# SZE-MOSZE-2020-Daemons
\ No newline at end of file
+# SZE-MOSZE-2020-Daemons
+
+After compiling, the game can be called with 6 parameters:</br>
+`./a.out <First player name> <First player Health> <First player Damage> <Second player name> <Second player Health> <Second player Damage>`
+</br>
+</br>
```

Comment:
jujj.

Reply:
Lecseréltük a break-et amenyiben ez volt a probléma.

Reply:
Hat md-be br-t irni kicsit fura, de ha mar irunk, akkor `<br>` :-)